README.md

```diff
@@ -76,7 +76,7 @@ See the [Getting Started](https://mathjs.org/docs/getting_started.html) for a mo
 
 ## Browser support
 
-Math.js works on any ES5 compatible JavaScript engine: node.js 4 or newer, Chrome, Firefox, Safari, Edge, and IE11.
+Math.js works on any ES5 compatible JavaScript engine: node.js 8 or newer, Chrome, Firefox, Safari, Edge, and IE11.
```

Comment:
Maybe having a numeric version of node here is asking for trouble and instead we should just say "node.js"

Reply:
ahh, very smart. Good point, will do.